src/main/java/org/spongepowered/api/stats/StatisticGroups.java

```diff
@@ -30,7 +30,14 @@
  */
 public class StatisticGroups {
 
+    /**
+     * Group for statistics that do not match in any group.
+     */
     public static final StatisticGroup GENERAL = null;
+    /**
+     * Statistic counting the distances moved in a specific way.
+     */
+    public static final StatisticGroup DISTANCE_MOVED = null;
```

Comment:
Its a completely arbitrary group which serves no purpose..., other than making the fact that it and the other 'general' statistics should be grouped some strange implementation detail.

you keep spouting that the general group is some arbitrary bin that I've cooked up and just chucked every other statistic but its really not...
![stats](http://puu.sh/gGtav/cba64b2b1b.png)